### ISSUE TITLE
Emesene Contact List

### ISSUE THREAD

**Issue by sttiretella** [octocat/Spoon-Knife#519]
I download emesene for windows but somehow  one of the contact i deleted on msn messenger, still  there on emesene.
While i am glad to see that person on my  list again  : ) how is that possible?
Which means this person didn't delete me?
not forget to mention while this person was on my emesene contact list but blocked. So unblocked...i see him but i don't know he sees me or not.
I also checked the option for -who has me  ..but seems inaccurate to me.

Another issue is
When i am writing on conversation box i see what i am writing but when i send it.some weird signs are coming out like a small boxed or something.
Why is that? any clue?
